x-pack/plugins/monitoring/server/lib/elasticsearch/ccr.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
+
+export function handleResponse(response) {
+  return get(response.defaults, 'xpack.ccr.enabled') === 'true';
+}
+
+export async function checkCcrEnabled(req) {
+  const { callWithRequest } = req.server.plugins.elasticsearch.getCluster('monitoring');
```

Comment:
Is this the right cluster to check against? Or is it `admin`? @pickypg

Reply:
Ah crap, of course! We can't just call this API against the production or monitoring clusters. What we need is to know via monitoring data if CCR is enabled or disabled on a particular production cluster. I'm not sure we have the data on whether CCR is enabled or not in `.monitoring-es-*`.

Reply:
We should start collecting this information in `cluster_stats` documents under `stack_stats.xpack.ccr.*` once https://github.com/elastic/elasticsearch/issues/37221 is resolved.